### ISSUE TITLE
Dialog boxes are poorly designed

### ISSUE THREAD

**Issue by jaigak** [UWPCommunity/Quarrel#500]
- You can't use the ESC key to exit a dialog box
- You can't use keyboard or controller to navigate (tab and arrow keys take effect behind the dialog box), only in a few cases
- Doesn't work with screen readers

**Comment by Waddle-Doo** [comment]
Can you elaborate on what you mean by "Dialog boxes"? Are you referring to subpages or flyouts?

**Comment by jaigak** [comment]
@Waddle-Doo Pages like the settings page that blurs the background.

**Comment by Waddle-Doo** [comment]
Okay, so you mean subpages! Cool 👍

**Comment by Waddle-Doo** [comment]
@Jaiganeshkumaran pinging you here to let you know about #378 as it may pique your interest as another major issue with subpages
